src/javascript/app/pages/user/account/settings/account_closure.js

```diff
@@ -0,0 +1,81 @@
+const BinarySocket = require('../../../../base/socket');
+const localize     = require('../../../../../_common/localize').localize;
+const Url          = require('../../../../../_common/url');
+
+const AccountClosure = (() => {
+    const form_selector = '#form_closure';
+    let $textField;
```

Comment:
variable names should be underscore separated, so `$text_field`, or better yet `$txt_field` to use a 3-letter input type